source/includes/documentation/_vue_js_elements.md

```diff
@@ -1896,19 +1896,61 @@ For examples of usage see: [JDI vuetify page tests for tabs](https://github.com/
     }
 ```
 
-Icon component provides a large set of graphial signs to provide context for various aspects of your application.
+__Icons__ - The `v-icon` component provides a large set of glyphs to provide context to various aspects of your application.
+For a list of all available icons, visit the official [Material Design Icons page](https://materialdesignicons.com/).
+To use any of these icons simply use the `mdi-` prefix followed by the icon name.
 
 ![Icons example](../../images/vuetify/icons.png)
-
-|Method | Description | Return Type
---- | --- | ---
-**is()** | Returns Assert class | IconAssert
-**hasType()** | Returns type of icon (mdi, svg, etc.) | String
-**hasColor()** | Returns color of icon in rgba | String
-**hasHeight()** | Returns height of icon | String
-**hasWidth()** | Returns width of icon | String
-
-For examples of usage see: [Vuetify Icon tests](https://github.com/jdi-testing/jdi-light/blob/vuetify-develop/jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/common/IconsTests.java).
+__Vuetify v2.6.14__ code example:
+```html
+<template>
+  <v-row justify="space-around">
+    <v-icon
+      large
+      color="green darken-2"
+    >
+      mdi-domain
+    </v-icon>
+
+    <v-icon
+      large
+      color="blue darken-2"
+    >
+      mdi-message-text
+    </v-icon>
+  </v-row>
+</template>
+```
```

Comment:
Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
![image](https://user-images.githubusercontent.com/55189259/212973786-e13f34e1-3d8e-4820-a16c-63fec23db0dd.png)